html-relative-links/projects/projects.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Projects</title>
+    <style>
+      img {
+        width: 300px;
+      }
+    </style>
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+
+      <li>
+        <a href="../about-me/about-me.html">About Me</a>
+      </li>
+
+      <li>
+        <a href="../projects.html">Projects</a>
```

Comment:
This should just be like this, because this file exist in the same directory, so we don't need to go up any levels

```suggestion
        <a href="projects.html">Projects</a>
```